StandbyControl.qml

```diff
@@ -229,6 +216,7 @@ Item {
 
     onModeChanged: {
         console.debug("Mode changed: " + mode);
+        // TODO create a device class for power saving instead of launching hard coded shell scripts from QML
```

Comment:
The whole standby control should be written in C++

Reply:
This will be implemented in #346 Hardware device abstraction. I didn't want to mix it with the wifi rewrite.